aip/general/0121.md

```diff
@@ -61,6 +61,10 @@ exposes a large number of resources with a small number of methods on each
 resource. The methods can be either the standard methods ([Get][], [List][],
 [Create][], [Update][], [Delete][]), or [custom methods][].
 
+For each standard operation, a field the represents the resource **must** use
```

Comment:
This would not apply to *Delete*, because the resource is never included: the method references the resource by `name` and then responds with `Empty` or an LRO. In turn, maybe say something that indicates, "where applicable"?

Reply:
Same thing for _Get_ really, it uses the `name` in the request, and then the actual resource as the response (which isn't a "field that represents the resource").

Reply:
I took another stab at a more principle-focused writing, so we don't have to go deep into edge cases like soft-delete APIs that need to return the resource.

PTAL.

Reply:
the formatting has changed a couple times, so I'll resolve the comment. PTAL at the new format.